### PR TITLE
Add git configuration options for BMO

### PR DESCRIPTION
**Pull request by Rozzii**
This change also adds logic to automatically save git metadata
to a metadata file.

### CODE REVIEW

File: 03_launch_mgmt_cluster.sh [hunk 1]
```diff
@@ -41,6 +42,8 @@ function clone_repo() {
     popd
     pushd "${REPO_PATH}"
     git checkout "${REPO_BRANCH}"
+    git checkout "${REPO_COMMIT}"
+    echo "${REPO_URL}:${REPO_BRANCH}:$(git rev-parse --short HEAD)" >> "${IPA_BUILD_METADATA_LOCATION}"
```
Comment (kashifest) [inline 03_launch_mgmt_cluster.sh:46]: Lets move this to Nordix/IPA building scripts

**Review by kashifest — CHANGES_REQUESTED** [review]
Lets move the IPA related work in Nordix

**Review by fmuyassarov — COMMENTED** [review]
/lgtm

**Comment by smoshiur1237** [comment]
Looks good to me

**Pull request merged**